src/mapeo-manager.js

```diff
@@ -658,6 +663,26 @@ export class MapeoManager extends TypedEmitter {
         )
       }
 
+      /** @type {"projectSettings"} */
+      const schemaName = 'projectSettings'
+
+      /** at-type import('@comapeo/schema').ProjectSettingsValue */
+      const settingsDoc = {
+        schemaName,
```

Comment:
accounting for the mistyped type annotation, can adjust this a bit:

```suggestion
      const settingsDoc = {
        schemaName: /** @type {const} */ ('projectSettings'),
```